Package.swift

```diff
@@ -233,6 +233,16 @@ let package = Package(
         .headerSearchPath("../../.."),
       ]
     ),
+    .target(
```

Comment:
Do we want to separate the CoreInternal APIs only for tests versus APIs needed by libraries into separate libraries?

In other words should this be FirebaseCoreInternalTestsSwift or part of FirebaseTestingSupport/Core?